### PR TITLE
Gokart

### PR DESCRIPTION
**Pull request by gejustin**
This adds support for [Gokart](https://github.com/praetorian-inc/gokart), a vulnerability scanning tool for Golang.

### CODE REVIEW

**Comment by EliSchleifer** [comment]
@gejustin - Looks like the gokart tests didn't run correctly. Let us know if you need help digging into why the test is not working correctly. @TylerJang27 for viz.

File: linters/gokart/plugin.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      package: github.com/praetorian-inc/gokart
+      environment:
+        - name: PATH
+          list: ["${linter}"]
+      run_from_root_file: go.mod
+      commands:
+        - name: lint
+          output: sarif
+          target: ${parent}
+          run: gokart scan -i ${workspace}/analyzers.yml -s ${target}
```
Comment (TylerJang27) [inline linters/gokart/plugin.yaml:16]: After investigation, you were correct. This should be:
`run: gokart scan -i analyzers.yml -s ${target}`

**Review by TylerJang27 — COMMENTED** [review]
Thanks for the contribution! Just need to fix the run command, as you suspected.

**Review by TylerJang27 — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**